projects/core/src/cart/event/cart.events.ts

```diff
@@ -26,11 +26,11 @@ export class CartAddEntrySuccessEvent extends CartEvent {
    * Event's type
    */
   static readonly type = 'CartAddEntrySuccessEvent';
-  productCode: string;
-  quantity: number;
-  entry: OrderEntry;
-  quantityAdded: number;
-  deliveryModeChanged: boolean;
+  productCode?: string;
+  quantity?: number;
```

Comment:
I think those 2 can remain mandatory, because we take them from the original AddToCart action (not from the response from backend). Is it right?